src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportGeneratorImplTest {
+    private static final Map<String, Integer> STORAGE = FruitStorage.fruits;
```

Comment:
i think you can just use it in your code directly, so you don't create this constant here

Reply:
✅